kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/aggregation/settings/tasks/KoverXmlReportTask.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.kover.gradle.aggregation.settings.tasks
+
+import kotlinx.kover.features.jvm.ClassFilters
+import kotlinx.kover.features.jvm.KoverLegacyFeatures
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.CacheableTask
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
+
+@CacheableTask
+internal abstract class KoverXmlReportTask : AbstractKoverTask() {
+    @get:OutputFile
+    internal abstract val reportFile: RegularFileProperty
+
+    @get:Input
+    abstract val title: Property<String>
```

Comment:
The title seems to be shared by both tasks, maybe move it to the abstract one?

Reply:
In addition to HTML and XML reports, there are others that lack a title, so I don't see the point in creating a class because of one property